package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@bcgsc-pori/graphkb-client",
-  "version": "4.3.2",
+  "version": "4.4.0",
   "lockfileVersion": 3,
   "requires": true,
   "packages": {
     "": {
       "name": "@bcgsc-pori/graphkb-client",
-      "version": "4.3.2",
+      "version": "4.4.0",
```

Comment:
minor thing: at line 16,  "@bcgsc-pori/graphkb-schema" should be "~4.0.0" instead of "4.0.0". This get set on npm install.

Reply:
It's not, but not a big deal.

Reply:
KBDEV-1385 to address package-lock.json file update